src/Mod/CAM/Path/Op/CircularHoleBase.py

```diff
@@ -25,6 +25,7 @@
 import Path
 import Path.Op.Base as PathOp
 import Path.Base.Drillable as Drillable
```

Comment:
Per the linting: `Use 'from Path.Base import Drillable' instead (consider-using-from-import)`